test/performance/odp_ipsecfwd.c

```diff
@@ -1339,13 +1360,13 @@ static void create_fwd_table_entry(config_setting_t *cfg, prog_config_t *config)
 		return;
 	}
 
-	entry = &config->fwd_entries[config->num_fwds];
+	mask = 0xFFFFFFFF << (IP_ADDR_LEN - mask);
```

Comment:
If mask length (it might be good to separate mask length variable from mask, for clarity) is 0, this triggers undefined behaviour, so that case probably needs to be special-cased.

Reply:
I keep forgetting the large enough bitshift resulting in undefined behavior instead of promotion. Will fix.

Reply:
V2.